types/manager-types.yaml

```diff
@@ -82,6 +82,7 @@ node_types:
             max_workers: 5
             remote_execution_port: 22
             user: centos
+            networks: { get_input: manager_networks }
```

Comment:
also needs to be added to `simple-manager-blueprint.yaml` because the `cloudify_agent` in there overwrites this, and this one is mostly ignored